article-analysis-project/article-processing.js

```diff
@@ -0,0 +1,44 @@
+//Modules
+const unfluff = require('unfluff');
+const cheerio = require('cheerio');
+const request = require('request');
+//Variables and Consts
+const url = 'https://abcnews.go.com/Politics/wireStory/canceling-ivanka-trump-spurs-backlash-kansas-college-71174430?cid=clicksource_4380645_2_heads_hero_live_headlines_hed'
+
+//Main
+
+request(url, (error, response, html) => {
+    if (!error && response.statusCode == 200){
+        console.log(html);
+        data = unfluff(html);
+        console.log(data.text);
+        console.log(data.author)
+        author = data.author
+        console.log(occurrences(data.text, " he ", false))
```

Comment:
This would have been caught by a linter, but it's good practice to not access functions / variables that have not been defined yet.  [this rule describes some of the context](https://eslint.org/docs/rules/no-use-before-define)